src/app/shared/refresh-organization/refresh-organization.component.html

```diff
@@ -20,7 +20,7 @@
   type="button"
   class="mr-1"
   data-cy="refreshOrganization"
-  (click)="refreshOrganization()"
+  (click)="openConfirmationDialog()"
```

Comment:
Not that we have very many of those types of orgs, and I don't know if it's easy to tell, but ideally we wouldn't have the dialog pop up with text talking about GitHub apps if it's a Bitbucket or Gitlab org.